src/containers/Tag/Tag.test.tsx

```diff
@@ -1,11 +1,91 @@
 import React from 'react';
-import { render, wait } from '@testing-library/react';
+import { render, wait, within, fireEvent, cleanup } from '@testing-library/react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { MockedProvider } from '@apollo/client/testing';
-import { GET_LANGUAGES, GET_TAG } from '../../graphql/queries/Tag';
+import { GET_LANGUAGES, GET_TAG, GET_TAGS_COUNT, FILTER_TAGS } from '../../graphql/queries/Tag';
+import { CREATE_TAG } from '../../graphql/mutations/Tag';
 import { Tag } from './Tag';
+import { Switch, Route } from 'react-router-dom';
+import { TagList } from './TagList/TagList';
 
+afterEach(cleanup);
 const mocks = [
+  {
+    request: {
+      query: CREATE_TAG,
+      variables: {
+        input: {
+          description: 'new Tag description',
+          isActive: false,
+          isReserved: false,
+          label: 'new Tag',
+          languageId: 1,
+        },
+      },
+    },
+    result: {
+      data: {
+        createTag: {
+          tag: {
+            description: 'new Tag description',
+            id: '121',
+            isActive: true,
+            isReserved: false,
+            label: 'new Tag',
+            language: {
+              id: '1',
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    request: {
+      query: GET_TAGS_COUNT,
+      variables: {
+        filter: {
+          label: '',
+        },
+      },
+    },
+    result: {
+      data: {
+        countTags: 2,
+      },
+    },
+  },
+  {
+    request: {
+      query: FILTER_TAGS,
+      variables: {
+        filter: {
+          label: '',
+        },
+        opts: {
+          limit: 10,
+          offset: 0,
+          order: 'ASC',
```

Comment:
Can you change this to DESC, so that it is consistent with the result.  I know it's mock data, but good to be correct.